seed-job/template.yaml

```diff
@@ -85,18 +85,14 @@ objects:
                         }
                     }
                     stage('Scan the image') {
-                        sh "docker run --name ${container_name} -d ${image_name} tail -f /dev/null"
+                        sh "docker create --name=scan-data 172.29.33.48:5000/nshaikh/ccp-jenkins-slave"
```

Comment:
Hard coding the container name like this will cause parallel scan requests to build. We must use a variable name. The `container_name` variable used earlier would suffice because OpenShift will make sure that parallel builds for *same* Jenkins Pipeline build don't happen.